### DISCUSSION TITLE
Cog Cross compilation

### DISCUSSION THREAD

**Discussion by gnulux** [Igalia/cog#513] [Q&A]
Hi, 

I'm not an expert with Meson or cMake build procedure and i'm wondering if you can provide a full step documentation in order to compile cog for aarc64  from x86_64 (Ubunutu) system.

B.regards

**Comment by philn** [comment]
This is not the right forum I'm afraid. You have a couple options like Buildroot or Yocto. For the latter some instructions are available in the meta-webkit wiki, https://github.com/Igalia/meta-webkit/wiki/WPE
